apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/businessrules/repository/WaRuleMetaDataRepository.java

```diff
@@ -0,0 +1,9 @@
+package gov.cdc.nbs.questionbank.businessrules.repository;
+
+import gov.cdc.nbs.questionbank.entity.businessRule.WaRuleMetadata;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface WaRuleMetaDataRepository extends JpaRepository<WaRuleMetadata, String> {
```

Comment:
The second parameter when extending `JpaRepository` is the type of the entity's Id.


The `WaRuleMetadata` Id is defined as a Long:
```java
@Id
@Column(name = "wa_rule_metadata_uid", nullable = false)
@GeneratedValue(strategy = GenerationType.IDENTITY)
private Long id;
```

so the repo should be:
```java
public interface WaRuleMetaDataRepository extends JpaRepository<WaRuleMetadata, Long> {
```